content/en/integrations/guide/_index.md

```diff
@@ -28,6 +28,8 @@ cascade:
     {{< nextlink href="integrations/guide/snmp-commonly-used-compatible-oids/" tag=" snmp" >}}SNMP commonly used and compatible OIDs{{< /nextlink >}}    
     {{< nextlink href="integrations/guide/versions-for-openmetrics-based-integrations" tag=" openmetrics" >}}Versioning for OpenMetrics-based integrations{{< /nextlink >}}     
     {{< nextlink href="integrations/guide/cloud-foundry-setup" tag=" pivotal cloud foundry" >}}Pivotal Cloud Foundry manual setup{{< /nextlink >}}     
+    {{< nextlink href="integrations/guide/application-monitoring-vmware-tanzu" tag=" VMWare Tanzu application monitoring" >}}Datadog Application Monitoring for VMware Tanzu{{< /nextlink >}} 
+    {{< nextlink href="integrations/guide/cluster-monitoring-vmware-tanzu" tag=" VMWare Tanzu cluster monitoring" >}}Datadog Cluster Monitoring for VMware Tanzu{{< /nextlink >}} 
```

Comment:
Should the tags for these just be `VMWare Tanzu`? I just wanted to check since the length of these stood out compared to the other tags on this page.

Reply:
That's a great callout, I had thought about that as well and agree that would read better.